db/migrate/20181201000100_create_message_recipients.foodsoft_messages.rb

```diff
@@ -0,0 +1,49 @@
+class CreateMessageRecipients < ActiveRecord::Migration
+  class Message < ActiveRecord::Base
+    has_many :message_recipients
+    has_many :recipients, through: :message_recipients, source: :user
+  end
```

Comment:
We'd also need to add the `MessageRecipient` class. I'm not sure if `Recipient` is necessary here, since we don't reference it below (might even be able to remove it here altogether, since we're dealing with ids only in the migration).